pallets/cash/src/internal/lock.rs

```diff
@@ -213,7 +132,7 @@ mod tests {
 
             assert_eq!(
                 lock_cash_principal_internal::<Test>(GEOFF, JARED, twice_principal_amount),
-                Err(Reason::RepayTooMuch)
+                Err(Reason::InsufficientChainCash)
```

Comment:
Fixed the name of these, I think `RepayTooMuch` was being awkwardly overused in this case.